tests/test_default/bad/empty.catala_en

```diff
@@ -19,7 +19,8 @@ $ catala Interpret -s A
   │          ‾
   └─ Article
 [ERROR]
-This variable evaluated to an empty term (no rule that defined it applied in this situation)
+This variable evaluated to an empty term (no rule that defined it applied in this situation): 
+error_empty ⟨ ⟨false ⊢ ⟨error_empty ∅⟩⟩ | false ⊢ ∅ ⟩
```

Comment:
Not sure you want to print a term here without the `--debug` option.

Reply:
Indeed, this looks like a debugging leftover